BTSPocket/Resources/Components/Alert/Alert.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Alert.swift
+//  BTSPocket
+//
+//  Created by bts on 01/04/21.
+//
+
+import Foundation
+import UIKit
+
+/// Show an OK alert just with title and message
+func showAlert(view:UIViewController, title: String, message: String) {
```

Comment:
this function can be included in an UIViewController+extension, create a swift file for this is unnecessary